exiftool.go

```diff
@@ -184,4 +184,14 @@ func Charset(charset string) func(*Exiftool) error {
 		e.extraInitArgs = append(e.extraInitArgs, "-charset", charset)
 		return nil
 	}
-}
\ No newline at end of file
+}
+
+// Function to use 'No print conversion' mode, see https://exiftool.org/exiftool_pod.html.
```

Comment:
The documentation does not fit godoc specifications. The comment should start with the name fonction.
https://blog.golang.org/godoc